packages/embeds/js/src/features/blocks/inputs/buttons/components/MultipleChoicesForm.tsx

```diff
@@ -42,17 +42,24 @@ export const MultipleChoicesForm = (props: Props) => {
     }
   };
 
-  const handleSubmit = () =>
+  const handleSubmit = () => {
+    const selectedItems = selectedItemIds().map((selectedItemId) =>
+      props.defaultItems.find((item) => item.id === selectedItemId),
+    );
     props.onSubmit({
       type: "text",
-      value: selectedItemIds()
-        .map(
-          (selectedItemId) =>
-            props.defaultItems.find((item) => item.id === selectedItemId)
-              ?.content,
-        )
+      value: selectedItems
+        .map((item) => {
+          return item?.value ?? item?.content ?? "";
+        })
+        .join(", "),
+      label: selectedItems
+        .map((item) => {
+          return item?.content ?? item?.value ?? "";
+        })
         .join(", "),
```

Comment:
I'd say, check if one of the selected items has an internal value, if that's the case, we set the label attribute otherwise we set it to undefined. Same for multiple picture choices